src/pages/MonthlyGood/MonthlyGoodLandingPage.vue

```diff
@@ -221,7 +221,7 @@ export default {
 			return this.heroContentGroup?.media?.[0]?.description ?? '';
 		},
 		heroContentGroup() {
-			return this.pageData?.page?.contentGroups?.homepageHero ?? null;
+			return this.pageData?.page?.contentGroups?.monthlygoodLandingHero ?? null;
```

Comment:
This is /monthlygood Unfortunately this was keyed off the type, but wasn't using the type component. Now it is keyed off the key, but this will require a removal of the type on this content group upon code release